### PR TITLE
Added autopep8 formatting.

### PR DESCRIPTION
**Pull request by genghisken**
Minor reformat of the python code using autopep8

### CODE REVIEW

**Review by stvoutsin — APPROVED** [review]
Looks good

**Review by esutorius — APPROVED** [review]

File: inflammation/serializers.py [hunk 1]
```diff
@@ -39,7 +40,8 @@ def deserialize(cls, data):
         instances = []
 
         for item in data:
-            item['observations'] = ObservationSerializer.deserialize(item.pop('observations'))
+            item['observations'] = ObservationSerializer.deserialize(
```
Comment (genghisken) [inline inflammation/serializers.py:43]: Not sure i like this style

**Review by genghisken — COMMENTED** [review]
Added a comment to relating to line 43 of serializers.py

**Review by bobmann — APPROVED** [review]
Looks fine to me

**Pull request merged**